mono_repo.yaml

```diff
@@ -1,8 +1,8 @@
+# Run `mono_repo generate --validate` to check 
+# that everything is up to date
+self_validate: true
+github:
```

Comment:
Add `cron` – good to run these weekly and catch regressions w/ the SDK dev branch

https://github.com/google/mono_repo.dart/blob/master/mono_repo.yaml#L8